packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -181,41 +181,29 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
 
             if (isTimestamp) {
                 return (
-                    <DateInput2
-                        className={disabled ? 'disabled-filter' : ''}
-                        placeholder={placeholder}
+                    <DateTimePicker
                         disabled={disabled}
-                        fill
-                        defaultTimezone="UTC"
-                        showTimezoneSelect={false}
-                        value={
-                            rule.values && rule.values.length > 0
-                                ? new Date(rule.values[0]).toString()
-                                : null
-                        }
-                        timePrecision={'millisecond'}
-                        formatDate={(value: Date) =>
-                            moment(value).format(`YYYY-MM-DD, HH:mm:ss:SSS`)
-                        }
-                        parseDate={(value) =>
-                            moment(value, `YYYY-MM-DD, HH:mm:ss:SSS`).toDate()
-                        }
-                        onChange={(value: string | null) => {
+                        placeholder={placeholder}
+                        withSeconds
```

Comment:
We let people set seconds?

Reply:
we even had milliseconds in BP -  I am removing milliseconds and leaving seconds. maybe one day we'll remove seconds too, who knows 🤷